bluesky/plan_stubs.py

```diff
@@ -214,7 +214,7 @@ def rel_set(obj, *args, group=None, wait=False, **kwargs):
         abs_set(obj, *args, group=group, wait=wait, **kwargs)))
 
 
-def mv(*args):
+def mv(*args, **kwargs):
```

Comment:
Should probably explicitly list `group` here and then do the `if is None...` dance or we will have 1 kwarg which will not work (and does make sense to broadcast across all of the motors.